app/mainAppWindow/index.js

```diff
@@ -155,6 +155,17 @@ function onDidFinishLoad() {
 	customCSS.onDidFinishLoad(window.webContents, config);
 }
 
+function onDidFrameFinishLoad(event, isMainFrame, frameProcessId, frameRoutingId) {
+	logger.debug('did-frame-finish-load', event, isMainFrame);
+
+	if (isMainFrame) {
+		return; // We want to insert CSS only into the Teams V2 content iframe
+	}
```

Comment:
I _think_ this should ensure we don't break the old Teams with this, but I can't really test that easily.